filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -331,6 +405,21 @@ function createContentItemNode($doc, $issue, $article, $galley, $objectLocalePre
 			}
 
 		}
+		//Adding Citation List (unstructured) on the content
+		$citationDao = DAORegistry::getDAO('CitationDAO'); /* @var $citationDao CitationDAO */
+		$parsedCitations = $citationDao->getBySubmissionId($article->getId())->toArray();
+		$citationListNode = $doc->createElementNS($deployment->getNamespace(), 'cl:CitationList');
+		$citationListNode->setAttributeNS('http://www.w3.org/2000/xmlns/' ,'xmlns:cl', 'http://www.medra.org/DOIMetadata/2.0/Citations');
+		foreach($parsedCitations as $citation) {
+			$articleCitationNode = $doc->createElementNS('http://www.medra.org/DOIMetadata/2.0/Citations', 'ArticleCitation');
+			$articleCitationNode->setAttribute('key', $article->getStoredPubId('doi') . '_ref' . $citation->getData('seq'));
+			$unstructuredCitationNode = $doc->createElementNS('http://www.medra.org/DOIMetadata/2.0/Citations', 'UnstructuredCitation');
```

Comment:
Why do we not also check if the citation has a DOI and consider it in the export?

Reply:
I don't know how to retrieve doi from Citation!? Could you please explain what do you mean?

Reply:
I think this is not relevant for the medra plugin. Just forget it. Sorry for the confusion... :-(